tests/e2e-tests/src/deployment-api.test.ts

```diff
@@ -14,20 +20,16 @@ describe('DeploymentApi', () => {
   let initialState: AiDeploymentList | undefined;
 
   beforeAll(async () => {
-    const queryResponse = await DeploymentApi.deploymentQuery(
-      {},
-      { 'AI-Resource-Group': resourceGroup }
-    ).execute();
+    const queryResponse = await getDeployments(resourceGroup);
     expect(queryResponse).toBeDefined();
     initialState = queryResponse;
-  });
+  }, 200000);
```

Comment:
[q] Why do we need a timeout here as well? Will `getDeployments` actually take 20 seconds?